### PR TITLE
add product-id to csproj

### PR DESCRIPTION
**Pull request by KojoBoat**

### CODE REVIEW

**Comment by bingenito** [comment]
Why isn't the target of this the main branch?

**Comment by KojoBoat** [comment]
> Why isn't the target of this the main branch?

We aren't ready for the main branch yet. The requirement was to set it up, in preparations to merge to the main before any push is done.

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -43,4 +43,6 @@ jobs:
       with:
         name: package
         path: package/*.nupkg
-        if-no-files-found: error   
\ No newline at end of file
+        if-no-files-found: error
+    # - name: Publish to nuget
```
Comment (bingenito) [inline .github/workflows/continuous-integration.yml:47]: These steps are being run on every CI build including PRs which would mean publishing a nuget.

Comment (KojoBoat) [reply .github/workflows/continuous-integration.yml:47]: It's actually commented out, but I'll be raising another PR to 'simulate' that and remove the commented job.

Comment (KojoBoat) [reply .github/workflows/continuous-integration.yml]: Hello @bingenito I've updated the PR.

**Review by bagdxk — APPROVED** [review]

**Pull request merged**